contracts/staking/test/catastrophe_test.ts

```diff
@@ -0,0 +1,89 @@
+import { ERC20ProxyContract, ERC20Wrapper } from '@0x/contracts-asset-proxy';
+import { DummyERC20TokenContract } from '@0x/contracts-erc20';
+import { blockchainTests, describe, expect, provider, web3Wrapper } from '@0x/contracts-test-utils';
```

Comment:
with blockchainTests we no longer need to import `provider` and `web3Wrapper`. we have these in the `BlockchainTestsEnvironment`